src/main/java/org/openbase/bco/bcozy/model/LanguageSelection.java

```diff
@@ -1,23 +1,24 @@
 /**
  * ==================================================================
- *
  * This file is part of org.openbase.bco.bcozy.
- *
+ * <p>
  * org.openbase.bco.bcozy is free software: you can redistribute it and modify
  * it under the terms of the GNU General Public License (Version 3)
  * as published by the Free Software Foundation.
- *
+ * <p>
```

Comment:
Just don't care about this anymore, the licence header will replaced soon anyway.